diem-node/src/lib.rs

```diff
@@ -48,28 +48,12 @@ pub struct DiemHandle {
     _rpc: Runtime,
     _mempool: Runtime,
     _state_synchronizer: StateSynchronizer,
-    network_runtimes: Vec<Runtime>,
+    _network_runtimes: Vec<Runtime>,
     _consensus_runtime: Option<Runtime>,
     _debug: NodeDebugService,
     _backup: Runtime,
 }
```

Comment:
What's the reason to keep around DiemHandle, if everything put into it is an unused variable?

Reply:
the `DiemHandle` is like an Arc; as long as you own it, the node will keep running. Drop it and it will shutdown.